v1Tov2Migrations/helper/createCapture.js

```diff
@@ -51,8 +51,8 @@ const createCapture = async (rawCapture, tree, trx, treeTags) => {
     note: rawCapture.note,
     attributes: rawCapture.extra_attributes,
     domain_specific_data: tree.domain_specific_data,
-    created_at: new Date().toISOString(),
-    updated_at: new Date().toISOString(),
+    created_at: rawCapture.captured_at,
+    updated_at: rawCapture.captured_at,
```

Comment:
I need the real time of the capture rather than the timestamp of running the migration script, it is useless for me.

Reply:
I think this created_at should be left, if you need the real time a better way would be to use joins with the field data and get the `captured_at` field

Reply:
I think this is fine to give the tree `created_at` the time of first `captured_at`, in real case,the tree should be created around the time of planting the tree. We can give tree `created_at` the time of doing the operation on admin panel, that's meaningful, but I don't think this is useful to give all the tree the same time of running the script.